identity/aziot-dps-client-async/src/lib.rs

```diff
@@ -306,6 +307,9 @@ impl Client {
         let body = hyper::body::to_bytes(body)
             .await
             .map_err(|err| std::io::Error::new(std::io::ErrorKind::Other, err))?;
+
+        // Unlike IoT Hub, DPS responses don't contain any potentially sensitive
```

Comment:
I'll eat my own words now looking at the response for TPM registration. DPS responses will contain the nonce, which is used as the signing key for registrations for a limited time period. So, let us remove the DPS response body too :)